.php_cs.dist

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env php
```

Comment:
I had this issue too with PhpStorm but I was able to force the language syntax by using "Associate with file Type" action. Any thought ?